flux-cli/src/main/java/com/marklogic/flux/impl/importdata/SplitterParams.java

```diff
@@ -129,6 +129,42 @@ public class SplitterParams implements SplitterOptions {
     )
     private String xmlNamespace;
 
+    @CommandLine.Option(
+        names = "--classifier-host",
+        description = "Hostname of the classifier service."
+    )
+    private String classifierHost;
+
+    @CommandLine.Option(
+        names = "--classifier-port",
+        description = "Port number of the classifier service."
+    )
+    private String classifierPort;
+
+    @CommandLine.Option(
+        names = "--classifier-protocol",
+        description = "Protocol (http/https) of the classifier service."
```

Comment:
This is fine for the initial demo, but given that it's a boolean value, I think `--classifier-https` (with no value) will be simpler. But no change needed now.